app/src/db/migrations/20231212000000_init.ts

```diff
@@ -142,6 +180,16 @@ export async function up(knex: Knex): Promise<void> {
           table.unique(['permitId', 'permitTypeId', 'submissionId']);
         })
       )
+      .then(() =>
+        knex.schema.raw(`create trigger before_insert_permit_trigger
+          before insert on public.permit
```

Comment:
just checking, did you want to set updatedAt for inserts to this table, or is it just updates?

Reply:
Yeah, needs to be set on insert for sorting purposes.